### PR TITLE
Add some missing Spanish translations

### PR DESCRIPTION
**Pull request by bayocr**
- Add some missing Spanish translations
- Fix fuzzy translations. 

I am new contributing to bookwyrm, if there is some wrong with the way I do it, please give some feedback and I will happy to apply it.

### CODE REVIEW

**Comment by mouse-reeve** [comment]
Thank you so much! These all look good to me, tagging @reesporte as well :)

File: locale/es/LC_MESSAGES/django.po [hunk 1]
```diff
@@ -1400,10 +1389,8 @@ msgid "Request an Invitation"
 msgstr "Solicitar una invitación"
 
 #: bookwyrm/templates/landing/layout.html:49
-#, fuzzy, python-format
-#| msgid "(Recommended if registration is open)"
 msgid "%(name)s registration is closed"
-msgstr "(Recomendado si la registración es abierta)"
+msgstr "El registro de %(name) está cerrado"
```
Comment (reesporte) [inline locale/es/LC_MESSAGES/django.po:1393]: one missing s!

**Comment by reesporte** [comment]
besides that, it looks good to me

**Comment by bayocr** [comment]
Thanks, it's fixed now.

**Comment by mouse-reeve** [comment]
Perfect :)

**Pull request merged**